### ISSUE TITLE
[enhancement] color settings

### ISSUE THREAD

**Issue by tolry** [TailorDev/Watson#94]
It would be nice to have some control over the color codes being used. I am having trouble reading tags on a windows machine using putty with default color settings (black background).

If I can help out make this happen, in any way, give me hint :smile:

**Comment by k4nar** [comment]
Yeah, I think we could add that to the config. Right now all the colors are listed here: https://github.com/TailorDev/Watson/blob/master/watson/cli.py#L32-L41

**Comment by tolry** [comment]
@k4nar thanks for your quick answer - I just 'patched' my code to make the tags readable, nice quick fix.

I could try making this configurable, but my Python skills are limited, to say the least. Should I give it a try or are there any coders already waiting to implement new features?

**Comment by Charc248** [comment]
I'm not sure if this is an issue w/ td-watson and instead w/ PuTTY 
I think the colour settings would be part of the terminal, and so instead part of PuTTY?
check out [this](http://www.siue.edu/~dbock/cmis565/PuTTYColorSettings.htm).

Not sure if adding colours on putty can be done  w/o setting PuTTY up properly :/

**Comment by jmaupetit** [comment]
@Charc248 this issue deals with [ANSI colors](https://en.wikipedia.org/wiki/ANSI_escape_code#Colors). Thanks to ANSI escape codes, your terminal is able to format displayed text. Then, your terminal (PuTTY in your case) is able to theme this display on top.

@SpotlightKid this issue should have been closed right?

**Comment by mapio** [comment]
Can the code owners please review https://github.com/TailorDev/Watson/pull/488 that will close this?

**Comment by webracer999** [comment]
On Powershell (blue background) "watson projects" isn't readable.
